src/foam/dao/DAO.js

```diff
@@ -95,6 +100,7 @@ foam.INTERFACE({
       async: true,
       type: 'FObject',
       swiftThrows: true,
+      documentation: 'Retrieves a single object from the DAO, whose id is equal to the parameter id. If the object is found, the promise resolves with the object. If the object is not found, it rejects with a foam.dao.ObjectNotFoundException.',
```

Comment:
No need to duplicate the description for find(), since then future updates will require updating in two spots and could get out of date. Better to just say this is the internal version of find().

Reply:
ok